### DISCUSSION TITLE
Who can mentor?

### DISCUSSION THREAD

**Discussion by The-Compiler** [pytest-dev/sprint#2] [General]
Looking at the [interest check](https://nuudel.digitalcourage.de/2tEsEpRcwMNcAXVO), I see many unknown names. That might just be me not being able to map real names to GitHub usernames (or maybe missing some of the more recent contributions), but it looks to me like we'll have a fair share of people without a deep involvement in pytest who are interested.

That's great! However, we'll also need enough people interested in mentoring someone for this to be a success...

@RonnyPfannschmidt @flub @asottile How is your situation looking like in the meantime?

Who else is coming and roughly knows their way around pytest internals? @okken? @obestwalter? Others?

**Comment by flub** [comment]
I expect to be around Tue & Wed for full-ish days, arriving sometime Mon
and leaving sometime Thu.  Happy to try and help mentor, though my
knowledge is rather ancient now...  I could be more flexible on dates if
needed as well, doesn't have to be that strict.

**Comment by asottile** [comment]
I am, unfortunately, going to be realistic with my commitments and going to have to skip this year :( I'm pretty sad about it but at the same time I just have too much going on with personal things, work, and such.
